othello/apps/games/tasks.py

```diff
@@ -31,9 +31,9 @@ def ping(game):
     return (timezone.now() - game.last_heartbeat).seconds < settings.CLIENT_HEARTBEAT_INTERVAL * 2
 
 
-def delete(game):
+def delete_game(game):
     if not game.is_tournament:
-        game.delete()
+        game.delete_game()
```

Comment:
It looks like you blindly replaced `delete(` with `delete_game(` in a bunch of places. I don't see a `Game.delete_game()` method; this will probably fail badly.

Reply:
argh I put too much trust in Pycharm refactoring

Reply:
This is not resolved. See https://github.com/tjcsl/othello-tourney/search?q=delete_game&unscoped_q=delete_game

Reply:
should be fixed now